doc/source/examples/dem/granuheap/granuheap.rst

```diff
@@ -0,0 +1,259 @@
+==================================
+Granuheap
+==================================
+
+This example simulates the flow of a cohesive granular material inside a Granuheap instrument using the discrete element method (DEM). For more detailed information on the concepts and physical meaning of the parameters in Lethe, we strongly recommend visiting `DEM parameters <../../../parameters/dem/dem.html>`_.
+
+
+----------------------------------
+Features
+----------------------------------
+
+- Solvers: ``lethe-particles``
+- Three-dimensional problem
+- Moving solid object
+- `GMSH <https://gmsh.info/>`_ grids
+
+
+----------------------------
+Files Used in This Example
+----------------------------
+
+All files mentioned below are located in the example's folder (``examples/dem/3d-granuheap``).
+
+- Mesh file: ``cylinder.msh`` and ``support.msh``
+- Parameters file for the problem: ``granuheap.prm``
```

Comment:
Did you add the .geo and the .msh files to your PR? They are not in the list of files added.

Reply:
@blaisb Yes, I added the two .msh files to the PR. I don't understand why they don't show on Git Hub. They are in the examples/dem/3d-granuheap folder and I have nothing to commit. I'll try to fix that.
I don't have .geo files. My Python code gives me the .msh file only. Maybe I can add the Python file in the example? or I can change my Python code to save the .geo files and add them to the example.

Reply:
Add the python file then :)